R/style_guides.R

```diff
@@ -75,7 +75,8 @@ tidyverse_style <- function(scope = "tokens",
         add_line_break_before_curly_closing,
       partial(
         set_line_break_after_opening_if_call_is_multi_line,
-        except_token = "COMMENT"
+        except_token_after = "COMMENT",
```

Comment:
Side note: I'm not sure about the `scope` argument to this function, maybe `scope` should be an argument of the top-level `style_()` functions? The default values of the arguments to this function here are useful, we can easily change that later.